internal/logs/fetcher_test.go

```diff
@@ -91,78 +91,78 @@ func fixPodWithContainers(containersName ...string) v1.Pod {
 	return pod
 }
 
-// fakePodGetter implements SIMPLY fake for getting pods and fetching associated logs
+// FakePodGetter implements SIMPLY fake for getting pods and fetching associated logs
 // Unfortunately needs to be implemented by us, because client-go fake client
 // in current implementation (client-go v1.12) returns zero-value `rest.Request`
 // and executing GetLogs(/*..*/).DoRaw() method, throws panic.
-type fakePodGetter struct {
+type FakePodGetter struct {
```

Comment:
Why is this type now being exported?
From what I can see, it is only used in this package.

Reply:
I've exported that because the constructor was public. 
But as you noticed it's used only in this pkg, so I've just changed the constructor `NewFakePodsGetter` to be  private.